lib/nose/plans/filter.rb

```diff
@@ -89,9 +89,9 @@ def update_state
         @state.eq -= @eq
         @state.cardinality *= @eq.map { |field| 1.0 / field.cardinality } \
                                  .inject(1.0, &:*)
-        return unless @range
+        return unless @ranges
 
-        @state.range = nil
+        @state.ranges = nil
         @state.cardinality *= 0.1
```

Comment:
Since there may be multiple range conditions, we may want to make this `0.1 ** @ranges.count`. This reflects the fact that we expect the cardinality to be even smaller if there are multiple range conditions.